components/Cards/SchoolCard/__strories__/SchoolCardBack.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
It might be better to drop the stories for the front and back school card, while just making sure the SchoolCard itself has a fleshed out story

Reply:
Never done these stories before, will take a look

Reply:
Ah, cool.

Okay, to clarify, I was suggesting deleting the `__stories__` folder for the front and back card.

You currently have code in `SchoolCardFront.stories.js` which I would consider perfect for `components/Cards/SchoolCard/__stories__/SchoolCard.stories.js`. So just have that one Story file, and you're good to go.

Also, really quick note: you've got a typo in the folder path: `__strories__` should be `__stories__`